packages/EasyCore/tests/Bridge/Symfony/Validator/Constraints/DateIntervalValidatorTest.php

```diff
@@ -0,0 +1,192 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyCore\Tests\Bridge\Symfony\Validator\Constraints;
+
+use EonX\EasyCore\Bridge\Symfony\Validator\Constraints\DateInterval;
+use EonX\EasyCore\Bridge\Symfony\Validator\Constraints\DateIntervalValidator;
+use EonX\EasyCore\Tests\Bridge\Symfony\AbstractSymfonyTestCase;
+use Mockery\MockInterface;
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\Context\ExecutionContextInterface;
+use Symfony\Component\Validator\Exception\UnexpectedTypeException;
+use Symfony\Component\Validator\Exception\UnexpectedValueException;
+use Symfony\Component\Validator\Violation\ConstraintViolationBuilderInterface;
+
+/**
+ * @covers \EonX\EasyCore\Bridge\Symfony\Validator\Constraints\DateIntervalValidator
+ */
+final class DateIntervalValidatorTest extends AbstractSymfonyTestCase
+{
+    /**
+     * @return string[][]
+     *
+     * @see testValidateFailsWithInvalidDateIntervalValues
+     */
+    public function provideInvalidDateIntervalValues(): array
+    {
+        return [
+            'Invalid date interval #1' => ['P4D1Y'],
+            'Invalid date interval #2' => ['6MT'],
+            'Invalid date interval #3' => ['abc'],
+            'Invalid date interval #4' => ['123'],
+        ];
+    }
+
+    /**
+     * @return mixed[]
+     *
+     * @see testValidateThrowsUnexpectedValueException
+     */
+    public function provideUnexpectedValues(): array
+    {
+        return [
+            'integer' => [53004085616, 'Expected argument of type "string", "integer" given'],
```

Comment:
I think we don't need a data provider for only one dataset.

Reply:
Agree. Deleted.